phoenix-core/src/test/java/org/apache/phoenix/coprocessor/TaskMetaDataEndpointTest.java

```diff
@@ -139,7 +140,12 @@ public RegionServerServices getRegionServerServices() {
             public ConcurrentMap<String, Object> getSharedData() {
                 return null;
             }
-        };
+
+                    @Override
```

Comment:
nit: Fix indentation.
Also, what is this change for?

Reply:
A previous version of this patch had PHOENIX-6155 but not its addendum that fixed compilation on certain HBase versions. The current version has the addendum and so this file shouldn't be altered in this patch anymore.